portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFields.testcase

```diff
@@ -663,4 +663,73 @@ definition {
 			fieldValidationErrorMessage = "This field is required.");
 	}
 
+	@description = "LPS-182981 - Verify if when adding a new option in a field this option is added only in the chosen field "
+	@priority = 4
+	test ViewOptionAddedOnlyInTheChosenField {
+		property portal.acceptance = "true";
+
+		task ("Given an created Form with a text field, a single selection field and a multiple selection field") {
+			FormsAdmin.addForm();
+
+			Form.editName(formName = "Form Test");
+
+			FormViewBuilder.addFieldByDoubleClick(fieldType = "Text");
+
+			Form.closeSidebar();
+
+			FormViewBuilder.addFieldByDoubleClick(fieldType = "Single Selection");
+
+			Form.closeSidebar();
+
+			FormViewBuilder.addFieldByDoubleClick(fieldType = "Multiple Selection");
+
+			Form.closeSidebar();
+		}
+
+		task ("and Given that created Form translated in another language") {
+			Form.addTranslation(localizedLanguage = "Portuguese (Brazil)");
+
+			Form.selectTranslation(localizedLanguage = "Portuguese (Brazil)");
+
+			Form.editName(formName = "Formulário Teste");
+
+			FormViewBuilder.editFieldLabel(
+				fieldLabel = "Text",
+				label = "Texto");
+
+			FormViewBuilder.editFieldLabel(
+				fieldLabel = "Single Selection",
+				label = "Seleção única");
+
+			FormViewBuilder.editFieldLabel(
+				fieldLabel = "Multiple Selection",
+				label = "Seleção múltipla");
+
+			Form.save();
+		}
```

Comment:
@alanafergs hey Alana! Can we update this test to only include what is necessary to cover the actual bug? Another way I would look at it is, "Will the bug occur if I miss these steps?"